docs/docs/Developing_with_JAC/Overview.md

```diff
@@ -9,26 +9,26 @@ Jaseci is a computational Model and language in one.Jaseci provides the ability
 
 ## Feature of Jaseci
 
-1. <strong>Easy to learn</strong> :
+1.<strong>Easy to learn: </strong> :
 Jaseci is a high level programming language. It is very easy to learn. The paradigm for Jaseci is based upon graphs.Once you underastand this new paradigm the language is simple.
 
-2. <strong>Free and Open Source </strong>
+2.<strong>Free and Open Source: </strong>
 Jaseci is completely free. Jaseci can be installled by runnning a simple pip command in your terminal.The Source code is avaliable here.
 
-3. <strong>Dynamically typed Language</strong> :
+3.<strong>Dynamically typed Language</strong> :
 Variable types are decided at run time so there is no need to sepcify them.
 
-4. <strong>Data Spacial</strong>
+4.<strong>Data Spacial: </strong>
 Allows for data spacial reasoning , problem solving and programming.
 
-5. <strong>Abstract Complexities ?</strong >
+5.<strong>Abstract Complexities: </strong >
 Abstract all complexities and allow for higher level programming. 
 
-6. <strong>Ecncapsulates sophiscated ecosystem of bleeding edge tools </strong>
+6.<strong>Ecncapsulates sophiscated ecosystem of bleeding edge tools: </strong>
  Advance AI models can be leveraged easily when uing Jaseci to make complex training and developemnt easier.
 
-7. <strong>New featues constantly being added</strong>
+7.<strong>New featues constantly being added: </strong>
 Extensible to include any new advance AI models , tools or libraries as they come out.
 
-8. <strong>Simple Serverside development</strong>
+8.<strong>Simple Serverside development: </strong>
 Server side development is made simple and easily scalable.
```

Comment:
Jaseci provides out-of-box production-grade containerization and orchestration so you can stand up a production-ready stack in minutes. With novel load balancing and facilitation techniques, your production Jaseci cluster scales intelligently with your application’s demand.